cmd/create/machinepool/machinepool.go

```diff
@@ -70,8 +70,13 @@ func addMachinePool(cmd *cobra.Command, clusterKey string, cluster *cmv1.Cluster
 			os.Exit(1)
 		}
 		if !isVersionCompatibleComputeSgIds {
+			formattedVersion, err := versions.FormatMajorMinorPatch(ocm.MinVersionForAdditionalComputeSecurityGroupIdsDay2)
+			if err != nil {
+				r.Reporter.Errorf("An error occurred formatting the version for output: %v", err)
+				os.Exit(1)
+			}
```

Comment:
Would it be wise to move these into a function as this code is used 3 times? I know its basically just moving a function call + error check into a function, but at the very least we should make the error string a constant/variable

Reply:
I agree with the constant, but having a wrapper around the error would mean returning an error as well and doing something similar as I don't think the os.Exit should happen within the function. Unfortunately it is more related to how golang currently deal with errors I think